### ISSUE TITLE
Whisper issue with offline players

### ISSUE THREAD

**Issue by llacczy** [elysium-project/server#1358]
(**TLDR**: pls make determinable when the whisper goes to offline player)

The current situation is, when a player goes offline the way "not intended", so alt+f4 or internet disconnect the character is still online.

(i believe its for reasons so ppl cant flee from combat or from pvp situation)

If i remember correctly the player character sits down when this happens and when online again the character will stand up and shines a bit.
So it is determinable when a player disconnects when you see him personal, but if you whisper to a player who is currently still ingame but disconnected, it is not determinable, you get no warning so you might think he got the message, but it is not the case.

(On the friendlist and on /who the disconnected player is still shown online)

I believe sending the information to the player trying to whisper an offline player would be really good if there is a way, not sure about how blizzlike these things are.

(**TLDR**: pls make determinable when the whisper goes to offline player)

**Comment by StadenElysium** [comment]
I believe there's room for improvement in how the online/offline status is handled when a player forcibly quits the client but in cases such as disconnections, there's often no way to know that the player is offline.

**Comment by Itembugs** [comment]
Yes i keep inviting people in battlegrounds and it shows them offline

or somewtimes i end up whispering to someone in a groupd they are offline but are talking to me because its only a visual offline bug or something, can this be fixed tytytytyty
